app/models/like.rb

```diff
@@ -0,0 +1,12 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  # Callbacks
+  after_save :update_post_likes_counter
+
+  # Methods
+  def update_post_likes_counter
+    post.update(likes_count: post.likes.count)
+  end
```

Comment:
- [ ] **[OPTIONAL]** I would recommend you make this method private in keeping with encapsulation and avoid other parts of the app from calling this method since it is local to this class.

Reply:
done